MCUIViewLayoutExample/UIViewLayoutExampleTests/MCUIViewLayoutPositionTest.m

```diff
@@ -331,4 +335,25 @@ - (void)testFitWidthWithMargins {
     XCTAssertTrue(rectEquals(110.0f, 1000.0f, 85.0f, 20.0f, rect), @"");
 }
 
+- (void)testCeilFloatToDisplayScale
+{
```

Comment:
Some bracket consistency ? Sometimes at the end of the line, sometimes on a new line.